docs/modules/ROOT/pages/references/component-parameters.adoc

```diff
@@ -67,34 +67,106 @@ reference:: xref:references/slo-parameters.adoc[See SLO parameters]
 This parameter allows configuring SLOs for AppCat.
 
 == `providers`
+Each provider's manifests are generated by a generic jsonnet function. So each of them have common parameters.
 
+=== `enabled`
+type:: boolean
+default:: `true|false`
+required:: true
 
-=== `providers.cloudscale`
-[horizontal]
-type:: dict
-reference:: xref:references/provider-cloudscale.adoc[See provider parameters]
+Specifies if this provider should be enabled or not.
 
+=== `apiVersion`
+type:: string
+default:: `N/A`
+required:: true
 
-This parameter allows configuring https://github.com/vshn/provider-cloudscale[the provider cloudscale].
+Define the apiVersion of the providers `providerConf` kind. Can vary for each provider.
 
+=== `connectionSecretNamespace`
+type:: string
+default:: `${appcat:crossplane:namespace}`
+required:: false
 
-=== `providers.exoscale`
-[horizontal]
-type:: dict
-reference:: xref:references/provider-exoscale.adoc[See provider parameters]
+Specify separate namespace where the provider saves the connection secrets.
 
+=== `namespace`
+type:: string
+default:: `${appcat:crossplane:namespace}`
+required:: true
 
-This parameter allows configuring https://github.com/vshn/provider-exoscale[the provider exoscale].
+Namespace where the provider should get installed.
 
+=== `credentials`
+type:: object
+default:: `null`
+required:: false
 
-=== `providers.kubernetes`
-[horizontal]
-type:: dict
-reference:: xref:references/provider-kubernetes.adoc[See provider parameters]
+Contents of the secret containing credentials to connect to the provider's external systems. E.g. another cloud or another Kubernetes cluster.
+There are some subfields that need to be specifed.
+
+==== `name`
+type:: string
+default:: `""`
+required:: true
+
+The name of the secret.
+
+==== `namespace`
+type:: string
+default:: `""`
+required:: true
+
+The namespace of the secret.
+
+==== `data`
+type:: map
+default:: `{}`
+required:: true
+
+The contents of the `data` map of the secret.
```

Comment:
I guess here you will have the details on how to connect to a different cluster? If yes can you elaborate more?

Reply:
This varies from provider to provider. So you'd have to consult the provider's docs.